packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -164,30 +149,21 @@ invoked at any time. Note that you should only call this function **once**.
-| [callbacks](#callbacks) | `Record<XEventName, (payload: XEventPayload<Event>, metadata: WireMetadata) => void` |          | A record of callbacks where the key is the event to listen, and the value is the callback to be executed whenever the event is emitted. For example, to listen to the `UserAcceptedAQuery` event: `{ UserAcceptedAQuery({ eventPayload }) { console.log('UserAcceptedAQuery', eventPayload); }`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
-| isSpa                   | `boolean`                                                                            |          | True when the X Components archetype is being run on top of an SPA.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| &lt;extra parameters>   | `any`                                                                                |          | Any other parameter to be sent directly to the API calls. For example, some times is needed to filter the search catalog with a warehouse parameter. In that case you can just add `warehouse: <your-warehouse-identifier>` to the snippet config.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+Check out the [X&nbsp;API](#x-api) section to learn more about the functions and parameters supported.
```

Comment:
You have removed the Required column in the new table, and it is much less clear.

Reply:
I've removed the _Required_ column to both reduce the length of the overall table and enlarge the width of the _Description_ column. Having too long descriptions makes it difficult to read. However, the information on which param is required or not is still available in the table. This information is not missing but met the same style used in the UI reference tables.  
Due to the huge size of descriptions, I've also relocated some additional information, by removing it from the table and placing it in callouts or other related sections. I've not removed nor deleted any important information, just rewritten or relocated it.